src/GraphQL/Types/Scalars/Enumeration/EnumValues.cs

```diff
@@ -0,0 +1,131 @@
+using System.Collections;
+using GraphQLParser;
+
+namespace GraphQL.Types
+{
+    public abstract class EnumValuesBase : IEnumerable<EnumValueDefinition>
```

Comment:
There's three classes in this file.  Let's move each to its own file, now that we have a whole folder for enumeration stuff.

Reply:
done